api/.env.local

```diff
@@ -4,6 +4,7 @@ MINTER_ACCOUNT_KEY_INDEX=0
 
 FUNGIBLE_TOKEN_ADDRESS=${FUNGIBLE_TOKEN_ADDRESS}
 NON_FUNGIBLE_TOKEN_ADDRESS=${NON_FUNGIBLE_TOKEN_ADDRESS}
+FUSD_TOKEN_ADDRESS=${FUSD_ADDRESS}
```

Comment:
Oh, is this to avoid a circular variable reference?

Is this `api/.env.local` file still needed? 🤔

Reply:
Hmm, yeah _maybe_ not.